core/src/main/java/com/novoda/noplayer/model/RendererCapability.java

```diff
@@ -0,0 +1,28 @@
+package com.novoda.noplayer.model;
+
+import com.google.android.exoplayer2.RendererCapabilities;
+
+public enum RendererCapability {
```

Comment:
I'd prefer if this was called `Support` tbh, it's the level of `support` that a given track has.

Reply:
it has this name internally, I've changed